website/docs/r/aws_auth_backend_client.html.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: "vault"
+page_title: "Vault: vault_aws_auth_backend_client resource"
+sidebar_current: "docs-vault-resource-aws-auth-backend-client"
+description: |-
+  Configures the client used by an AWS Auth Backend in Vault.
+---
+
+# vault\_aws\_auth\_backend\_client
+
+Configures the client used by an AWS Auth Backend in Vault.
```

Comment:
Nit: is it helpful to say `the HTTP client` here? 🤔

Reply:
Opinion: The Vault docs never reference it as an HTTP client, so I think it's better without. But am open to changing it if you feel strongly one way or another.

Reply:
I'm not super-familiar with what's going on behind the scenes here, but I suggest generally that we try to use the same terminology used in the Vault documentation so it's easier for users to understand how these things relate. If Vault is considering HTTP to be an implementation detail here, then it seems right for us to do that too.